benchexec/tablegenerator/util.py

```diff
@@ -97,17 +97,18 @@ def open_url_seekable(path_url, mode='rt'):
         return io.TextIOWrapper(response)
 
 
-def split_number_and_unit(s):
+def split_number_and_unit(s, reverse=False):
```

Comment:
Adding a parameter `reverse` to this method might technically make sense, but not semantically. For example, `reverse` could also mean "recombine number and unit". What this method is now used for has nothing to do with "units".

A more general interface should be designed that allows both use cases, or they should be kept separate.